examples/torch_tensorrt/torchcompile/T5/README.md

```diff
@@ -0,0 +1,56 @@
+# TorchServe inference with torch.compile with tensorrt backend
+
+This example shows how to run TorchServe inference with T5 [Torch-TensorRT](https://github.com/pytorch/TensorRT) model
+
+
+
+[T5](https://huggingface.co/docs/transformers/en/model_doc/t5#inference) is an encode-decoder model used for a variety of text tasks out of the box by prepending a different text corresponding to each task. In this example, we use T5 for translation from English to German.
+
+### Pre-requisites
+
+- Verified to be working with `torch-tensorrt==2.3.0`
+Installation instructions can be found in [pytorch/TensorRT](https://github.com/pytorch/TensorRT)
+
+Change directory to examples directory `cd examples/torch_tensorrt/T5/torchcompile`
```

Comment:
Should we revert the order and do examples/torch_compile/tensor_rt instead? Then we could move all examples/pt2 into there too so people find everything about compile in a single place? I assume there will be no other integration point for TRT in the future other than compile.

Reply:
For tensorrt, onnx, we actually want the starting point to be tensorrt, onnx, since customers are looking specifically for these. There are github issues where customers are still looking for an onnx example. Will add an example on onnx next